types/fsp.go

```diff
@@ -38,8 +38,10 @@ func CheckFsp(fsp int) (int8, error) {
 	if fsp == int(UnspecifiedFsp) {
 		return DefaultFsp, nil
 	}
-	if fsp < int(MinFsp) || fsp > int(MaxFsp) {
+	if fsp < int(MinFsp) {
```

Comment:
Have you checked all the functions which invoke CheckFsp to make sure that they are all compatible with MySQL?

Reply:
Yes, I haven't found any compatible problems yet.